### PR TITLE
Fix design

### PR DESCRIPTION
**Pull request by cedrtang**

### CODE REVIEW

File: app/src/main/java/org/eclipse/keyple/demo/control/activities/CardContentActivity.kt [hunk 1]
```diff
@@ -33,7 +36,7 @@ class CardContentActivity : AppCompatActivity() {
         setContentView(R.layout.activity_card_content)
         setSupportActionBar(findViewById(R.id.toolbar))
 
-        findViewById<Button>(R.id.present_btn).setOnClickListener {
+        findViewById<Button>(R.id.presentBtn).setOnClickListener {
```
Comment (Yamapps) [inline app/src/main/java/org/eclipse/keyple/demo/control/activities/CardContentActivity.kt:39]: call directly 'presentBtn.setOnClickListener'

File: app/src/main/java/org/eclipse/keyple/demo/control/activities/NetworkInvalidActivity.kt [hunk 2]
```diff
@@ -24,11 +25,12 @@ class NetworkInvalidActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_network_invalid)
         setSupportActionBar(findViewById(R.id.toolbar))
+        toolbarLogo.setImageResource(R.drawable.ic_logo_white)
 
         val cardContent: CardReaderResponse? = intent.getParcelableExtra(CARD_CONTENT)
         findViewById<TextView>(R.id.invalid_description).text =
             String.format(getString(R.string.card_invalid_desc), cardContent?.cardType?: getString(R.string.card_invalid_default))
-        findViewById<Button>(R.id.present_btn).setOnClickListener {
+        findViewById<Button>(R.id.presentBtn).setOnClickListener {
```
Comment (Yamapps) [inline app/src/main/java/org/eclipse/keyple/demo/control/activities/NetworkInvalidActivity.kt:33]: idem

File: app/src/main/java/org/eclipse/keyple/demo/control/activities/SplashScreenActivity.kt [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        // Make sure this is before calling super.onCreate
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splashscreen)
+
+        /*
+         * Wait for Wizway Device to be connected
+         */Timer().schedule(object : TimerTask() {
+            override fun run() {
```
Comment (Yamapps) [inline app/src/main/java/org/eclipse/keyple/demo/control/activities/SplashScreenActivity.kt:33]: No need for a timer here, Wizway will not be integrated to this app

**Review by Yamapps — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/eclipse/keyple/demo/control/activities/SplashScreenActivity.kt (hunk 3 above)
Comment (cedrtang) [reply app/src/main/java/org/eclipse/keyple/demo/control/activities/SplashScreenActivity.kt:33]: Maybe not for wizway but the splashscreen must be displayed some time to finish the animation, that's the point of a splashscreen. You can play with the timer as you see fit

**Pull request merged**